Hey, can I add my component into the mainline code?

dwellons opened the pull request.

dwellons committed:
generated my addition component

dwellons committed:
Added a progress spinner for fun

had to add import(s) to app.module.ts

dwellons committed:
ready for week 7 slack n tell demo

remember to import the controls! app.module.ts

dwellons committed:
week-07-slack-n-tell done

wasn't as hard as I thought

dwellons committed:
fixed wording for clarity

dwellons committed:
Merge branch 'master' into dwellons/week-07-slack-n-tell

tsteelematc commented:
You have merge conflicts to resolve... There's a vid on how to do this, but the general steps are...

. fetch latest master
. change to your PR branch
. merge master into your PR branch
. resolve merge conflicts, preserving the previous components and putting yours at the bottom
. Then push your PR branch back out to GitHub

Hope this makes sense. Ask questions if you have them...

dwellons committed:
Merge with Master Branch

Merged master into week 07, added components from classmates and mine to the end

dwellons commented:
@tsteelematc I just finished with the videos and locally it says that I am up to date with master, I don't see anything on this page though. Will I have to close this request and create a new one? Thank you

tsteelematc commented on src/app/app.component.html:
```diff
@@ -1,6 +1,9 @@
+<h1>Components App</h1>
+<app-pizza-toppings></app-pizza-toppings>
+
 <app-pizza-toppings></app-pizza-toppings>
```
Probably don't want 2 pizza toppings component : - O

Otherwise it looks like you've resolved conflict and added yours to the bottom, nice work!

Take out one of the extra's and I can merge yours next...

tsteelematc replied:
<img width="1038" alt="image" src="https://github.com/user-attachments/assets/fc7eade1-f2fa-46c2-bedf-59c268108e15">

dwellons replied:
definitely didn't want two of those!

dwellons committed:
Removed extra component

removed extra pizza toppings component <app-pizza-toppings></app-pizza-toppings> from the app.component.html file.
